prow/config/tide.go

```diff
@@ -24,6 +24,8 @@ import (
 	"k8s.io/test-infra/prow/github"
 )
 
+const timeFormatISO8601 = "2006-01-02T15:04:05Z"
```

Comment:
GitHub [might work](https://developer.github.com/v3/#explicitly-provide-an-iso-8601-timestamp-with-timezone-information) with time.RFC3339 too (same as 8601, but also has TZ offset at the end): RFC3339

Reply:
Tested: nope!